retroarch.c

```diff
@@ -3191,7 +3193,10 @@ static enum runloop_state runloop_check_state(
             current_input, RARCH_MOVIE_RECORD_TOGGLE);
 
       if (pressed && !old_pressed)
+      {
+         command_event(CMD_EVENT_RECORD_INIT, NULL);
```

Comment:
This is a hack. Basically this is what instances record_init.
A new hotkey should be defined for this. Currently it's using the bsv recording hotkey.

Also it it's pressed while recording is enabled it should call deinit instead.